### PR TITLE
Removed outdated License Info

### PR DESCRIPTION
**Pull request by DJPlaya**
Ref #1086

### CODE REVIEW

**Comment by Headline** [comment]
@dvander should we bump license version/date or is this fine as is?

**Comment by DJPlaya** [comment]
My mistake, its important to change that too ^^

File: licenses/LICENSE.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SOURCEMOD LICENSE INFORMATION
-VERSION: JUNE-13-2008
+VERSION: 29.9.2019
```
Comment (dvander) [inline licenses/LICENSE.txt]: If you want to bump the version, might as well keep it in the same format (or YYYY.MM.DD which looks nicer)

**Comment by asherkin** [comment]
I can't do a line comment on it as it is outside of the changed lines (WTF GitHub), but the exceptions list includes a mention of the JIT that can be removed as well.

It also looks like the last word got lost at some point, it should read "All trademarks are property of their respective owners in the US and other countries."

File: licenses/LICENSE.txt (hunk 1 above)
Comment (DJPlaya) [reply licenses/LICENSE.txt]: Ime using the [most often used format](https://en.wikipedia.org/wiki/Date_format_by_country) DMY, or is there a default format for this repo?

**Comment by DJPlaya** [comment]
@asherkin 
The last word was missing before, dident noticed ^^

You mean changing:
"As a special exception, AlliedModders LLC gives you permission to link the code
of this program (as well as its derivative works) to "Half-Life 2," the "Source 
Engine," the "SourcePawn JIT," and any Game MODs that run on software by the 
Valve Corporation. "
to
"As a special exception, AlliedModders LLC gives you permission to link the code
of this program (as well as its derivative works) to "Half-Life 2," the "Source 
Engine and any Game MODs that run on software by the 
Valve Corporation. "
?

File: licenses/LICENSE.txt (hunk 1 above)
Comment (dvander) [reply licenses/LICENSE.txt]: Well, if you're gonna change the formatting here, YYYY-MM-DD looks better. I think that's ISO standard.

Comment (DJPlaya) [reply licenses/LICENSE.txt]: Ok, dident know SM is using ISO 8601

**Comment by Headline** [comment]
@DJPlaya yes that's what he meant

**Pull request merged**